README.md

```diff
@@ -13,7 +13,7 @@ An [ember-cli](http://www.ember-cli.com) addon for using [Materialize](http://ma
 
 **Ember 2.0 Friendly**
 
-**Materialize Version ~0.97.0**
+**Materialize Version 0.97.5**
```

Comment:
need to imply somehow that 0.97.6 is not supported. is this ok?
**Materialize Version <= 0.97.5**